src/App.tsx

```diff
@@ -1,19 +1,24 @@
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import Counter from './components/Counter/Counter';
-import Header from './components/Header/Header';
+// import Header from './components/Header/Header';
 import GenreSelect from './components/GenreSelect/GenreSelect';
 import './App.css';
 import MovieTitle from './components/MovieTitle/MovieTitle';
 import MovieDetails from './components/MovieDetails/MovieDetails';
 import SortControl from './components/SortControl/SortControl';
 
 function App() {
-  const genreList: Genre[] = ['Horror', 'Romantic', 'Thrilling', 'Comedy'];
+  const genreList: Genre[] = ['All','Horror', 'Romantic', 'Thrilling', 'Comedy'];
   const movieSortOptions: movieSortOption[] = ['Release Date', 'Title'];
   const [selectedGenre, setSelectedGenre] = useState(genreList[0]);
   const [selectedSortOption, setSelectedSortOption] = useState(
     movieSortOptions[0]
   );
+  useEffect(()=>{
```

Comment:
Nope! You dont need effect here. The default value needs to be setted in the compnent itself